src/libraries/System.IO.Packaging/src/System/IO/Packaging/PackageXmlStringTable.cs

```diff
@@ -127,39 +127,24 @@ internal static NameTable NameTable
 
         // Nested Types
         [StructLayout(LayoutKind.Sequential)]
-        private struct XmlStringTableStruct
+        private readonly struct XmlStringTableStruct
         {
             private readonly object _nameString;
             private readonly PackageXmlEnum _namespace;
             private readonly string? _valueType;
+
             internal XmlStringTableStruct(object nameString, PackageXmlEnum ns, string? valueType)
             {
                 _nameString = nameString;
                 _namespace = ns;
                 _valueType = valueType;
             }
 
-            internal object Name
-            {
-                get
-                {
-                    return (string)_nameString;
-                }
-            }
-            internal PackageXmlEnum Namespace
-            {
-                get
-                {
-                    return _namespace;
-                }
-            }
-            internal string? ValueType
-            {
-                get
-                {
-                    return _valueType;
-                }
-            }
+            internal object Name => (string)_nameString;
```

Comment:
Should the backing field just be `string _nameString` and the cast done once up front to reduce the need for repeated cast checks each time the property is retrieved?

Reply:
Yeah, this code makes little sense :) It's always being passed a string but is accepting and storing it as an object.  Then the Name property is casting it to a string only to have it be cast back to object immediately.  Will fix, thanks.